### ISSUE TITLE
CSS Vendor Prefixes

### ISSUE THREAD

**Issue by iamfrntdv** [styled-components/vue-styled-components#28]
I think there is an issue with css vendor prefixes. In example webkit prefixes are generated for text-transform. I think there is no need to do that.

**Comment by iamfrntdv** [comment]
Is there any way to control what vendor prefixes to generate?

**Comment by dwightjack** [comment]
Hi coming to this issue a bit late.

Since v1.1.1 styled-components switched from postcss to [stylis](https://github.com/thysultan/stylis.js) as CSS preprocessor. The issue you encountered looks like a bug in its side; in fact they fixed it in [v3.1.8](https://github.com/thysultan/stylis.js/issues/38). 

`vue-styled-components` still depends on v2.x but since the main project [already moved to v3.x](https://github.com/styled-components/styled-components/blob/master/package.json#L70), it should be safe to update it in your project by explicitly install `stylis` as dependency (`npm install stylis@3.x --save`).

Anyway, should you need any fine grained control on prefixes generation, you should be able to control it by configuring the library or by creating a custom plugins.

To achieve that, import the library in your entry point and configure it before you define any styled-component. Something like:

```js
import styled from 'vue-styled-components';
import stylis from 'stylis';

stylis.use({
  prefix: (key, value, context) => {
    if (key === 'my-unprefixed-property') return false;
  }
});

const StyledTitle = styled.h1`
  text-transform: uppercase;
`;
```

I'm not 100% sure about this solution though, because the docs aren't that clear about applying prefix plugins...

That said, I add a gentle reminder to @liqueflies to update the project's dependencies 😃

**Comment by liqueflies** [comment]
Done here: #85 

Thank you.
